src/CMakeLists.txt

```diff
@@ -0,0 +1,179 @@
+# recurse to subdirs
+# --------------------------------------------------------------------
+add_subdirectory(libxc)
```

Comment:
Where do you define ${libxc}?

Reply:
I don't define it anywhere.  `libxc` is a string literal on this line (keep in mind, CMake will interpret any text as a string literal, even if it's not quoted).

Reply:
So `libxc` is equal to `${CMAKE_CURRENT_SOURCE_DIR}/libxc` here?

Reply:
yep